app/lib/features/pins/pages/pin_page.dart

```diff
@@ -119,117 +115,69 @@ class PinPage extends ConsumerWidget {
+                    Icons.chevron_left,
+                    size: 42,
                   ),
-                );
-              }
-              if (pin.hasFormattedText()) {
-                content.add(RenderHtml(text: pin.contentFormatted() ?? ''));
-              } else {
-                final text = pin.contentText();
-                if (text != null) {
-                  content.add(Text(text));
-                }
-              }
-
-              return SliverToBoxAdapter(
-                child: Padding(
-                  padding: const EdgeInsets.all(15),
-                  child: Card(
-                    child: Column(
-                      mainAxisAlignment: MainAxisAlignment.start,
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        ListTile(
-                          key: Key(
-                            pin.eventIdStr(),
-                          ), // FIXME: causes crashes in ffigen
-                          leading: Icon(
-                            isLink
-                                ? Atlas.link_chain_thin
-                                : Atlas.document_thin,
-                          ),
-                          title: Text(pin.title()),
-                          subtitle: Padding(
-                            padding: const EdgeInsets.only(top: 4),
-                            child: Wrap(
-                              alignment: WrapAlignment.start,
-                              children: [SpaceChip(spaceId: spaceId)],
-                            ),
-                          ),
-                          trailing: IconButton(
-                            onPressed: () => showAdaptiveDialog(
-                              context: context,
-                              builder: (ctx) => ReportContentWidget(
-                                title: 'Report this Pin',
-                                description:
-                                    'Report this content to your homeserver administrator. Please note that your adminstrator won\'t be able to read or view files, if space is encrypted',
-                                eventId: pinId,
-                                roomId: pin.roomIdStr(),
-                                senderId: pin.sender().toString(),
-                                isSpace: true,
-                              ),
-                            ),
-                            icon: Icon(
-                              Atlas.warning_thin,
-                              color: Theme.of(context).colorScheme.error,
-                            ),
-                          ),
-                        ),
-                        Padding(
-                          padding: const EdgeInsets.all(8),
-                          child: Column(children: content),
-                        ),
-                      ],
-                    ),
+                ),
+                title: TextFormField(
```

Comment:
why always make that a text form field? Doesn't that mean it won't overflow for text larger than the width? Why not just ...

```suggestion
                title: !pinEdit.editMode ?Text(acterPin.title()` :  TextFormField(
```